### PR TITLE
CSD-278 Styled number list

### PR DESCRIPTION
**Pull request by rebeccahongsf**
# READY FOR REVIEW

# Summary
- Styled number list

# Needed By (Date)
- 08/24

# Urgency
- medium

# Steps to Test
1. Pull this branch
2. Clear cache
3. Create a number list and apply "Number List" styles from dropdown. Compare results to [Figma](https://www.figma.com/file/CpblbZldiGM6oYawa9aB4Q/Look-%26-Feel?node-id=1162%3A541)

# Associated Issues and/or People
- [CSD-278](https://stanfordits.atlassian.net/browse/CSD-278)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

**Comment by rebeccahongsf** [comment]
Oh wait a minute I see now — I need to import the styles within the `ckeditor.scss` as well!

**Comment by rebeccahongsf** [comment]
The issues should be resolved now 👍  Styles should now appear in WYSIWYG and page

File: config/sync/filter.format.stanford_html.yml [hunk 1]
```diff
@@ -1,109 +0,0 @@
-uuid: e2f5c93d-fb67-4955-9e71-3c07efdf1ef9
```
Comment (pookmish) [inline config/sync/filter.format.stanford_html.yml:1]: We need this file

Comment (rebeccahongsf) [reply config/sync/filter.format.stanford_html.yml]: 😬  And it has been brought back to life

**Pull request merged**